appinfok8s.go

```diff
@@ -120,3 +124,16 @@ func (ai AppengineInfoK8s) NumInstances(moduleName, version string) (int, error)
 		return instances, nil
 	}
 }
+
+func (ai AppengineInfoK8s) GcpServiceAccountName(saName string) (string, error) {
+	namespace := ai.NativeProjectID()
+	if sa, err := ai.clientset.CoreV1().ServiceAccounts(namespace).Get(ai.c, saName, metav1.GetOptions{}); err != nil {
```

Comment:
It's possible we could have multiple services accounts using workload identity in the namespace, right? Wondering if we could pull the service account name from the pod then look up that SA's annotation

Reply:
We could, but I think we should probably hold off on that until we need it and just pass it into this function for now. since this repo is open source I didn't want to expose the name directly in here; hence why i'm choosing to pass it in for now

Reply:
nvm, just implemented the pod lookup for the service account name